tests/queries/0_stateless/02811_csv_input_empty_as_default_for_nullable_string.reference

```diff
@@ -0,0 +1,3 @@
+a1	b1		d1
```

Comment:
I don't understand. So, here "empty as default" for Nullable(String) should yield NULL.

It looks like, you've changed the correct behavior with the incorrect one.